lib/model/CProbabilityAndInfluenceCalculator.cc

```diff
@@ -49,6 +49,9 @@ using TProbabilityCalculation2Vec = core::CSmallVector<maths_t::EProbabilityCalc
 using TSizeDoublePr = std::pair<std::size_t, double>;
 using TSizeDoublePr1Vec = core::CSmallVector<TSizeDoublePr, 1>;
 
+const std::string SINGLE_BUCKET_FEATURE_LABEL{"single_bucket"};
+const std::string MULTI_BUCKET_FEATURE_LABEL{"multi_bucket"};
```

Comment:
I think we'd be better off defining these once as static constants in `maths::CModel`. As it is these have to match the locally declared strings in `CTimeSeriesModel.cc`.

Better still would be to use an enum and update the relevant types to be defined w.r.t. this enum.